Source/Csla.Shared/Reflection/ServiceProviderMethodCaller.cs

```diff
@@ -48,7 +50,8 @@ public static System.Reflection.MethodInfo FindDataPortalMethod(object target, T
     /// <param name="targetType">Type of domain object</param>
     /// <param name="attributeType">Data portal operation attribute</param>
     /// <param name="criteria">Data portal criteria values</param>
-    public static System.Reflection.MethodInfo FindDataPortalMethod(Type targetType, Type attributeType, object[] criteria)
+    public static System.Reflection.MethodInfo FindDataPortalMethod<T>(Type targetType, object[] criteria)
+      where T : DataPortalOperationAttribute
     {
       if (targetType == null)
         throw new ArgumentNullException("targetType");
```

Comment:
Use `nameof(targetType)` here

Reply:
When throwing the exception? But the value is null so has no name.